bootstrap.d/app-editors.yml

```diff
@@ -5,12 +5,14 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://git.savannah.gnu.org/git/nano.git'
-      tag: 'v5.8'
-      version: '5.8'
+      tag: 'v5.9'
+      version: '5.9'
       tools_required:
         - host-autoconf-v2.69
         - host-automake-v1.15
         - host-pkg-config
+        - virtual: pkgconfig-for-target
```

Comment:
As this seems uncommon, the question for me is whether adding this is necessary?